plugin/NERD_commenter.vim

```diff
@@ -172,7 +172,7 @@ let s:delimiterMap = {
     \ 'exports': { 'left': '#' },
     \ 'factor': { 'left': '! ', 'leftAlt': '!# ' },
     \ 'fancy': { 'left': '#' },
-    \ 'faust': { 'left': '//' }, 
+    \ 'faust': { 'left': '//' },
```

Comment:
Please leave whitespace cleanup for a separate commit, otherwise `git blame` starts giving useless results. If in the future something comes up about faust and I want to know what it is, what put it there, why we used the delimiters we  did, etc, I will `git blame` this file to find who contributed it and what issuer or PR it was discussed in. Mucking around through unrelated commits is no fun. Thanks.

Reply:
Sure. This must have being done from my vim, it automatically trims whitespace. But will surely revert it